rotkehlchen/tests/api/test_ethereum_transactions.py

```diff
@@ -980,6 +980,7 @@ def query_transactions(rotki) -> None:
         'notes': 'Burned 0.00863351371344 ETH for gas',
         'sequence_index': 0,
         'timestamp': 1642802807,
+        'extra_data': None,
```

Comment:
add a transaction with extra data and test that its extra data are properly serialized